EXAMPLES.md

```diff
@@ -188,6 +188,27 @@ await client.loginWithPopup({
 });
 ```
 
+### Switch to a different organizations
+
+When working with multiple organizations, there might be a situation where you want your users to be able to switch between different organizations.
+
+Important to realize is that the authentication state in the SDK is build around the organization, so in order to switch organizations, you need to logout and login again. However, it's enough to log out from the SDK and not from Auth0. Doing that ensures that, when calling login again, the user might not be prompted for their credentials depending on whether or not they have an active session with Auth0.
+
+```ts
+async function switchOrganization(newOrganization: string) {
+  await client.logout({ openUrl: false });
+  await client.loginWithRedirect({
+    authorizationParams: {
+      organization: newOrganization
+    }
+  });
+}
+```
+
+**Note:** Ensure to pass any additional parameters to `loginWithRedirect` (or `loginWithPopup`) just as you might have passed on other occurences of calling login.
+
+Doing the above will ensure the SDK removes any token that is linked to the current organization, and logs in again to retrieve tokens for the new organization.
```

Comment:
You don't nee this paragraph, plus it reads like it's referring to the "Note" rather than the code snippet